### ISSUE TITLE
Please require/support newer illuminate/console

### ISSUE THREAD

**Issue by bakroistvan** [rocketeers/rocketeer#674]
I am facing difficulties to resolve the following dependency conflicts:
````
"require" :
    {
        "php": ">=5.3",
        "robmorgan/phinx": "^0.5.4"
    },
    
    "require-dev" :
    {
        "anahkiasen/rocketeer": "^2.2"
    }
````

Is it possible to require `"illuminate/console": "~5.2"` instead of `"illuminate/console": "~4.2"`

**Comment by Antoniossss** [comment]
This is supposed to be resolved by v3.0 but it looks like project died. Check https://github.com/rocketeers/rocketeer/issues/542

**Comment by Anahkiasen** [comment]
Should be good on `develop` – Rocketeer 3 does not use `illuminate/console` anymore (it uses `symfony/console` directly). It also is not made anymore to be required within your project and is instead meant to be used globally (through the PHAR or `composer global require`) like other deployments tools
